releasenotes/notes/sampling-vqe-and-qaoa-ecfb36a0a300f69b.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    A :class:`~qiskit.algorithms.minimum_eigensolvers.SamplingVQE` class is introduced, which is
+    optimized for diagonal hamiltonians and leverages a ``sampler`` primitive. A
+    :class:`~qiskit.algorithms.minimum_eigensolvers.QAOA` class is also added that subclasses 
+    ``SamplingVQE``.
+    
```

Comment:
Do you think we should add a small sample code block here at least for SamplingVQE ?